airmozilla/search/templates/search/home.html

```diff
@@ -92,6 +92,7 @@ <h5 class="entry-upcoming">
 
           <a class="go" href="{{ href }}">{{ _('See more') }}</a>
         </p>
+        <a class="star{% if star_on %} star-on{% endif %}" data-id="{{ event.id }}" title="Save as a starred event"></a>
```

Comment:
That `{% if star_on %}` would that ever be true? 
Isn't that just for when you display events on the starred page?